_episodes_rmd/13-dplyr.Rmd

```diff
@@ -263,6 +263,35 @@ gdp_pop_bycontinents_byyear <- gapminder %>%
               sd_gdp_billion=sd(gdp_billion))
 ```
 
+### Connect mutate with logical filtering: ifelse
```

Comment:
All section headers should be level 2 (even if they are subsections), because of how the template expects things. So please change this H3 to a H2